libdnf/transaction/rpm_package.cpp

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright (C) 2017-2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "libdnf/transaction/rpm_package.hpp"
+
+#include "libdnf/rpm/nevra.hpp"
+#include "libdnf/transaction/db/rpm.hpp"
+#include "libdnf/transaction/transaction.hpp"
+
+#include <algorithm>
+#include <map>
+#include <sstream>
+
+
+namespace libdnf::transaction {
+
+
+Package::Package(Transaction & trans) : TransactionItem::TransactionItem(trans, Type::RPM) {}
+
+
+uint32_t Package::get_epoch_int() const {
+    if (get_epoch().empty()) {
+        return 0;
+    }
+    return std::stoi(get_epoch());
+}
+
+
+std::string Package::to_string() const {
+    return libdnf::rpm::to_nevra_string(*this);
+}
+
+
+
+/*
```

Comment:
Please could you add TODO to describe plans for the code/comment?

Reply:
It's in the hpp already.